cmd/swagger/commands/generate/spec.go

```diff
@@ -1,20 +1,3 @@
-//go:build !go1.11
-// +build !go1.11
-
-// Copyright 2015 go-swagger maintainers
```

Comment:
Keep licensing?

Reply:
ok. Good point. The newer version omitted the licensing terms. Fixed